JustSaying/IPublishConfiguration.cs

```diff
@@ -1,11 +1,15 @@
-using System.Collections.Generic;
-
-namespace JustSaying
-{
-    public interface IPublishConfiguration
-    {
-        int PublishFailureReAttempts { get; set; }
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using JustSaying.AwsTools.MessageHandling;
+
+namespace JustSaying
+{
+    public interface IPublishConfiguration
+    {
+        int PublishFailureReAttempts { get; set; }
         int PublishFailureBackoffMilliseconds { get; set; }
-        IReadOnlyCollection<string> AdditionalSubscriberAccounts { get; set; }
-    }
+        Action<MessageResponse> ResponseLogger { get; set; }

```

Comment:
Interested to know thoughts on Martin's question around either making this async or having an async version (considering JustSaying has a synchronous and asynchronous publish call).